src/manage/workspaces/index.md

```diff
@@ -10,9 +10,7 @@ One workspace can contain multiple projects. The only limit is the storage quota
 
 Users can be invited to a workspace as [members or guests](../permissions/). These users don't need to have their own subscription. 
 
-::: tip
-Want to read more about workspaces and why they were introduced to <MainPlatformName />? Visit our blog [Introducing Workspaces: Simplified Collaboration](https://merginmaps.com/blog/introducing-workspaces-simplified-collaboration).
```

Comment:
I would leave the link to the blog post here as well - sometimes is easier to read abot a topic than to watch a video.